integration/cli/build.configequality.test.ts

```diff
@@ -76,95 +74,104 @@ describe("build", () => {
             expect(res).toBeDefined();
-          }
-        });
+          },
+        );
 
+        // --- ClusterRoleBinding tests ---
         it.each(sources)(
           "ClusterRoleBinding in $source resources should have correct roleRef and subjects",
           ({ getResources }) => {
-            const clusterRoleBinding = resource.select(
-              getResources(),
-              kind.ClusterRoleBinding,
-              peprUuid,
-            );
-            expect(clusterRoleBinding).toBeDefined();
-            expect(clusterRoleBinding.roleRef.name).toBe(peprUuid);
-            expect(clusterRoleBinding.subjects?.at(0)?.name).toBe(peprUuid);
+            const crb = resource.select(getResources(), kind.ClusterRoleBinding, peprUuid);
+            expect(crb).toBeDefined();
+            expect(crb.roleRef.name).toBe(peprUuid);
+            expect(crb.subjects?.[0]?.name).toBe(peprUuid);
           },
         );
 
+        // --- RoleBinding tests ---
         it.each(sources)(
           "RoleBinding in $source resources should have correct roleRef and subjects",
           ({ getResources }) => {
-            const roleBindingStore = resource.select(
-              getResources(),
-              kind.RoleBinding,
-              `${peprUuid}-store`,
-            );
-            expect(roleBindingStore).toBeDefined();
-            expect(roleBindingStore.roleRef.name).toBe(`${peprUuid}-store`);
-            expect(roleBindingStore.subjects?.at(0)?.name).toBe(`${peprUuid}-store`);
+            const rb = resource.select(getResources(), kind.RoleBinding, `${peprUuid}-store`);
+            expect(rb).toBeDefined();
+            expect(rb.roleRef.name).toBe(`${peprUuid}-store`);
+            expect(rb.subjects?.[0]?.name).toBe(`${peprUuid}-store`);
           },
         );
 
+        // --- Deployment (admission) ---
         it.each(sources)(
           "Deployment (admission) in $source resources should have correct labels and volumes",
-          // eslint-disable-next-line complexity
           ({ getResources }) => {
-            const deploy = resource.select(getResources(), kind.Deployment, peprUuid);
-            expect(deploy).toBeDefined();
-            expect(deploy.metadata?.labels?.app).toBe(peprUuid);
-            expect(deploy.metadata?.labels?.["pepr.dev/uuid"]).toBe(moduleConfig.uuid);
-            expect(deploy.spec?.template?.spec?.serviceAccountName).toBe(peprUuid);
-            expect(
-              deploy.spec?.template?.spec?.volumes?.find(v => v.name === "tls-certs")?.secret
-                ?.secretName,
-            ).toBe(`${peprUuid}-tls`);
+            const d = resource.select(getResources(), kind.Deployment, peprUuid);
+            expect(d).toBeDefined();
+            expect(d.metadata?.labels?.app).toBe(peprUuid);
+            expect(d.metadata?.labels?.["pepr.dev/uuid"]).toBe(moduleConfig.uuid);
+            expect(d.spec?.template?.spec?.serviceAccountName).toBe(peprUuid);
```

Comment:
```suggestion
            const deployment = resource.select(getResources(), kind.Deployment, peprUuid);
            expect(deployment).toBeDefined();
            expect(deployment.metadata?.labels?.app).toBe(peprUuid);
            expect(deployment.metadata?.labels?.["pepr.dev/uuid"]).toBe(moduleConfig.uuid);
            expect(deployment.spec?.template?.spec?.serviceAccountName).toBe(peprUuid);
```

Preference for variable names that are not one character long